lib/team_api/canonicalizer.rb

```diff
@@ -108,7 +110,7 @@ def self.hyphenate_yyyymmdd(timestamp)
     # different strings for the same thing.
     def self.canonicalize_tag_category(tags_xrefs)
       return if tags_xrefs.nil? || tags_xrefs.empty?
-      tags_xrefs.replace(CrossReferencer.map_reduce(tags_xrefs.values,
+      tags_xrefs.replace(LambdaMapReduce.map_reduce(tags_xrefs.values,
```

Comment:
this might be easier to read if args are split up:
```ruby
tags_xrefs.replace(
  LambdaMapReduce.map_reduce(
    tags_xrefs.values,
    ->(xref) { [[xref['slug'], xref]] },
    method(:consolidate_xrefs)).to_h
  )
)
```

```